frontend/app/scripts/controllers/trainingCtrl.js

```diff
@@ -35,6 +41,22 @@ angular.module('frontendApp')
         });
       };
 
+      $scope.openWaitList = function() {
+        ngDialog.open({
+          template: "views/popups/waitList.html",
+          controller: 'WaitListCtrl',
+          resolve: {
+            participants: function() {
+              return {
+                participants: $scope.training.participants,
```

Comment:
You can use for resolve only $scope.training object. There is no need to transmit every property individually.